content/docs/reference-react-dom-server.md

```diff
@@ -61,15 +61,15 @@ If you plan to use React on the client to make the markup interactive, do not us
 ReactDOMServer.renderToNodeStream(element)
 ```
 
-Render a React element to its initial HTML. Returns a [Readable stream](https://nodejs.org/api/stream.html#stream_readable_streams) that outputs an HTML string. The HTML output by this stream is exactly equal to what [`ReactDOMServer.renderToString`](#rendertostring) would return. You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
+Renderiza um elemento React para seu HTML inicial. Retorna um [Stream legível](https://nodejs.org/api/stream.html#stream_readable_streams) que gera uma string HTML. A saída HTML desse stream é exatamente igual ao que [`ReactDOMServer.renderToString`](#rendertostring) retornaria. Você pode usar este método para gerar HTML no servidor e enviar o markup para o request inicial para ter carregamentos de página rápidos e para permitir que motores de pesquisa rastreiem suas páginas para fins de SEO.
```

Comment:
```suggestion
Renderiza um elemento React para seu HTML inicial. Retorna uma [Stream legível](https://nodejs.org/api/stream.html#stream_readable_streams) que gera uma string HTML. A saída HTML dessa stream é exatamente igual ao que [`ReactDOMServer.renderToString`](#rendertostring) retornaria. Você pode usar este método para gerar HTML no servidor e enviar o markup para o request inicial para ter carregamentos de página rápidos e para permitir que motores de pesquisa rastreiem suas páginas para fins de SEO.
```

in my option stream is a female object, I can be wrong.

Reply:
Hmm... I don't agree here. I think 'Stream' is 'Fluxo' and that's male. But we'll wait for another reviewer.

Reply:
```suggestion
Renderiza um elemento React para seu HTML inicial. Retorna um [Stream legível](https://nodejs.org/api/stream.html#stream_readable_streams) que gera uma string HTML. A saída HTML desse stream é exatamente igual ao que [`ReactDOMServer.renderToString`](#rendertostring) retornaria. Você pode usar este método para gerar HTML no servidor e enviar o markup no request inicial para ter carregamentos de página mais rápidos e para permitir que motores de pesquisa rastreiem suas páginas para fins de SEO.
```

Reply:
I'm not sure about that, what do you think about this one @glaucia86 and @cezaraugusto?

Reply:
I know that this is already approved but I would suggest a few changes. 

First, I don't thin "Readable Streams" should be translated. It's a full term, translating it would prevent people from finding it in online searches. 

Also "A saída HTML [...] é exatamente igual **à**" once both are female nouns (as saídas) 

And I guess "motores de busca" are more standardized than "motores de pesquisa".

Reply:
thanks for the feedback @halian-vilela. could you please create a pull-request with those changes?

Reply:
> thanks for the feedback @halian-vilela. could you please create a pull-request with those changes?

Sure! Done: https://github.com/reactjs/pt-BR.reactjs.org/pull/35

Reply:
> I know that this is already approved but I would suggest a few changes.
> 
> First, I don't think `Readable Streams` should be translated. It's a full term, translating it would prevent people from finding it in online searches.
> 
> Also "A saída HTML [...] é exatamente igual **à**" once both are female nouns (as saídas)
> 
> And I guess "motores de busca" are more standardized than "motores de pesquisa".

Good catch! 👌